packages/compass-indexes/src/utils/fetch-reason.ts

```diff
@@ -0,0 +1,16 @@
+export enum FetchReasons {
+  /**
+   *
```

Comment:
I think those are pretty self explanatory, but highlighting just in case you wanted to add comments but forgot 🙂

Reply:
Forgot :)